components/webcompat_reporter/ui/components/basic/structure.ts

```diff
@@ -4,18 +4,18 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import * as React from 'react'
-import styled from 'brave-ui/theme'
+import styled from 'styled-components'
 import Button, { Props as ButtonProps } from 'brave-ui/components/buttonsIndicators/button'
 
-export const SideBySideButtons = styled<{}, 'div'>('div')`
+export const SideBySideButtons = styled('div')<{}>`
```

Comment:
It can, yes, and it will still get correctly-typed theme props. But we can do it in a follow-up. This was the result of a find/replace just switching the order and removing the duplication.